### DISCUSSION TITLE
How/Where to subscribe to a Domain Event?

### DISCUSSION THREAD

**Discussion by TCrosmanDSI** [jasontaylordev/CleanArchitecture#497] [Q&A]
How do you subscribe to these domain events?

Say I wanted to send an email out upon a "TodoItemCompletedEvent", where/how can this be done?

Any simple examples?

Thanks!

**Comment by ronnieholm** [comment]
A few simple examples are here:

https://github.com/jasontaylordev/CleanArchitecture/tree/main/src/Application/TodoItems/EventHandlers
